packages/backend-plugin-api/src/services/definitions/HttpRouterService.ts

```diff
@@ -22,11 +22,16 @@ export interface HttpRouterServiceAuthPolicy {
   allow: 'unauthenticated' | 'user-cookie';
 }
 
+/** @public */
+export interface HttpRouterHealthCheckConfig {
+  handler: () => Promise<any>;
+}
+
 /**
  * @public
  */
 export interface HttpRouterService {
```

Comment:
What do you think about keeping health checks at the root level instead? Something like `/.backstage/health/v1/readiness` for checking whether the entire instance is up, and `/.backstage/health/v1/<plugin-id>/readiness` for individual plugins?

Reply:
yeah nice!

Reply:
but the other way around, `/.backstage/health/v1/readiness/<plugin-id>`